pyseir/inference/model_fitter.py

```diff
@@ -680,6 +680,40 @@ def run_for_fips(cls, fips, n_retries=3):
         return model_fitter
 
 
+def _execute_model_for_fips(fips):
+    if fips:
+        model_fitter = ModelFitter.run_for_fips(fips)
+        return model_fitter
+    return None
+
+def _persist_results_per_state(state_df):
+        county_output_file = get_run_artifact_path(state_df.fips[0], RunArtifact.MLE_FIT_RESULT)
+        data = state_df.drop(['state', 'mle_model'], axis=1)
+        with open(county_output_file, 'w') as buf:
+            data.to_json(buf)
+
+        for fips, county_series in state_df.iterrows():
+            with open(get_run_artifact_path(fips, RunArtifact.MLE_FIT_MODEL), 'wb') as f:
+                pickle.dump(county_series.mle_model, f)
+
+
+def build_county_list(state):
+    """
+    Build the and return the fips list
```

Comment:
I think it might be helpful to put a little more context about the whitelist here.  I don't really know a ton about the whitelist and probably wouldn't expect it to be summoned here given the name

Reply:
Actually, i'm not exactly clear on how the whitelist is working either, but noted cc @erccarls

Reply:
Whitelist is generated in `inference/whitelist_generator.py`. It is a straightforward cut on a few things like min total cases, min total deaths and how many "new XXX" datapoints exist above a threshold.

The reason it is called here is because we do not want to run inference over those counties with very little data.